docs/src/main/tut/docs/settings.md

```diff
@@ -68,13 +68,13 @@ micrositeHomepage := "https://47deg.github.io/sbt-microsites/"
 micrositeOrganizationHomepage := "http://www.47deg.com"
 ```
 
-- `micrositeTwitter`: this setting is used for the Twitter integration. This generate a meta that Twitter use in his [Twitter Cards](https://cards-dev.twitter.com/validator).
+- `micrositeTwitter`: this setting is used for the Twitter integration. This generate a meta that Twitter use in its [Twitter Cards](https://cards-dev.twitter.com/validator).
```

Comment:
Still `This generate` instead of `This generates`
AND

` Twitter use` instead of ` Twitter uses`